src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyPushOnlyView.js

```diff
@@ -0,0 +1,53 @@
+import { createButton, loc } from 'okta';
+import { BaseForm } from '../../internals';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+
+const Body = BaseForm.extend(Object.assign({
+  noButtonBar: true,
+
+  initialize() {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    this.addView();
+  },
+
+  title() {
+    return loc('oie.okta_verify.push.title', 'login');
+  },
+
+  addView() {
+    this.add(createButton({
+      className: 'button button-wide button-primary send-push',
+      title: loc('oie.okta_verify.sendPushButton', 'login'),
+      click: () => {
+        this.$el.submit();
+      },
+    }));
+    this.add(
+      `<span class='accessibility-text' role='alert'>${loc('oie.okta_verify.sendPushButton', 'login')}</span>`,
+    );
+  },
+
+  render() {
+    BaseForm.prototype.render.apply(this, arguments);
+    this.$el.addClass('okta-verify-push-challenge');
```

Comment:
I think we can add the class directly to the form with `className: 'okta-verify-push-challenge',` (probably around like 6 or so, right below `BaseForm.extends`